impress-org/givewp-next-gen
# Feature: style core donation form block

## Description



This is the initial styling of our new donation form block.  We're keeping it simple right now by just selecting a form but want that experience to look polished.  As we progress with the functionality of this block, more styling updates will occur.

## Affects



The donation form block

## Visuals



![2022-07-29 14 27 43](https://user-images.githubusercontent.com/10138447/181822276-9dd3db68-0801-4700-8a01-9b446e1e2e08.gif)


## Testing Instructions



- Run `npm install && npm run dev`
- Make sure you have a couple next gen forms added.
- Go to a gutenberg enabled page and add the GiveWP next gen block
- Select a form and view the front-end.

## Pre-review Checklist
